__tests__/app.test.js

```diff
@@ -115,3 +115,54 @@ describe("GET /api/users", () => {
       });
   });
 });
+
+describe.only("PATCH /api/articles/:article_id", () => {
+  test("returns status 200 and the updated restaurant object with updated votes amount when successful", () => {
+    return request(app)
+      .patch("/api/articles/1")
+      .expect(200)
+      .send({ inc_votes: -100 })
+      .then(({ body: article }) => {
```

Comment:
You could use nested destructuring here but I wouldn't recast body as article as the article is a key on the body